GitUI/CommandsDialogs/FormReflog.cs

```diff
@@ -174,7 +175,11 @@ private void gridReflog_MouseMove(object sender, MouseEventArgs e)
 
             if (hit.Type == DataGridViewHitTestType.Cell && _lastHitRowIndex != hit.RowIndex)
             {
-                gridReflog.Rows[_lastHitRowIndex].Selected = false;
+                if (_lastHitRowIndex < gridReflog.Rows.Count)
+                {
+                    gridReflog.Rows[_lastHitRowIndex].Selected = false;
+                }
+
                 _lastHitRowIndex = hit.RowIndex;
                 gridReflog.Rows[_lastHitRowIndex].Selected = true;
```

Comment:
Are we sure `_lastHitRowIndex` here is within the array bounds?

Reply:
We are never sure with software but we can reasonably think that yes and that Winforms return the row index that is under the mouse... That works well at the moment.